gc/base/ParallelHeapWalker.cpp

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
```

Comment:
For all of the new files in the GC folder directly are they newly written code or are they being donated by the J9 project?